src/Strategy.java

```diff
@@ -0,0 +1,22 @@
+import java.util.LinkedHashMap;
+
+public abstract class Strategy {
+      abstract LinkedHashMap<Integer, Integer> giveMoney(LinkedHashMap<Integer, Integer> billsBox, int amount);
+
+      /**
+       * @param billsBoxBefore array contains information about different ATM bills before operation
+       * @param billsBoxAfter  newArray array contains information about different ATM bills after operation
+       * @return string the contains information about how much and what nominal where given
+       */
+      static String makeGiveReport(LinkedHashMap<Integer, Integer> billsBoxBefore, LinkedHashMap<Integer, Integer> billsBoxAfter) {
```

Comment:
Do you need this here? It looks like a util method or similar.
In this case you can convert the abstract class to the interface which is preferable here